blazor/mention/code-snippet/dynamic-object-binding.razor

```diff
@@ -0,0 +1,60 @@
+@using Syncfusion.Blazor.DropDowns
+@using System.Dynamic
+
+<SfMention TItem="DynamicDictionary" DataSource="@Orders" suggestionCount=6>
+    <TargetComponent>
```

Comment:
Don't use the SuggestionCount property for all code snippets.. Add the specific section alone

Reply:
How much data's available in remote data?

Reply:
Addressed